MeetupPlatformApi/Authentication/AuthenticationManager.cs

```diff
@@ -21,27 +21,43 @@ public CurrentUserInfo GetCurrentUserInfo(ClaimsPrincipal user)
         var idClaim = user.Claims.Single(claim => claim.Type == ClaimTypes.NameIdentifier);
         var id = Guid.Parse(idClaim.Value);
 
-        return new() {UserId = id};
+        return new() { UserId = id };
     }
 
-    public string IssueAccessToken(UserEntity user) =>
-        IssueToken(
+    public TokenPair IssueTokenPair(UserEntity user)
+    {
+        var accessToken = IssueAccessToken(
             payload: new Dictionary<string, object>
-            {
-                {ClaimTypes.NameIdentifier, user.Id}
-            },
-            lifetime: configuration.AccessTokenLifetime);
+                {
```

Comment:
Why did You change indentation level here? Now `payload` parameter name is on it's proper place, but all following rows are shifted to the right. Please, return previously used indentation.